test/suites/osconfig/lifecycle.robot

```diff
@@ -20,11 +20,10 @@ Restarting MicroShift In The Middle Of Startup Should Succeed
     ...    (for example due to not stopped microshift-etcd.scope).
     [Template]    Restart MicroShift ${TIME_TO_WAIT} Seconds After Starting
 
-    40
-    30
-    20
-    15
-    10
+    29
```

Comment:
I was under impression we'd have 23, 29, 37 - not drop the 40s one.
Honestly, there are no good or bad values, only not enough of them :D 
How much time do we gain by doing this?

Reply:
Here's how much time each iteration takes. If we add one more timeout, it'll exceed the 20m "limit".
Another thing we may want to do it to reduce the number of iterations from 3 to 2- wdyt?
![image](https://github.com/openshift/microshift/assets/85498834/ac7df3c9-b733-48fc-9ac3-009a89c774a6)

Reply:
I don't know, I don't like this imposed 20m limit we're imposing on ourselves. We should strive towards testing as comprehensively as possible without taking too many shortcuts. If the robustness/stability test needs more time, we should run it as such hammering it as much as possible.

Reply:
Maybe, we should move this test to periodics then?

Reply:
I removed this change from the PR until we reach an agreement. Anyway, it does not exceed 20m runtime by far.

Reply:
> Maybe, we should move this test to periodics then?

It does run as periodic in composer tests. It's definitely not a presubmit test material